### PR TITLE
pre-cfp information, logos, dates and images

### PR DESCRIPTION
**Pull request by connor21**
I've prepared the pages with all information available at this stage.

### CODE REVIEW

File: site/content/events/2016-kiel/_sponsors.txt [hunk 1]
```diff
@@ -16,29 +16,41 @@ end
...
+<% @hsponsors = [
+{ :image => "kiwi.png", :name => "Kieler Wirtschaftsförderung GmbH", :link => "http://www.kiwi-kiel.de/"},
 ] %>
-<%= make_sponsor_links(@ssponsors) %>
+<%= make_sponsor_links(@hsponsors) %>
 <br />
-<% if @ssponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a>
-<% elsif @ssponsors.size < 6 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a><% end %>
+<% if @hsponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a>
+<% elsif @hsponsors.size < 1 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a><% end %>
```
Comment (phrawzty) [inline site/content/events/2016-kiel/_sponsors.txt]: Do you need more than one Host Sponsor?

Comment (connor21) [inline site/content/events/2016-kiel/_sponsors.txt]: We have one Host Sponsor who provides the venue. No other Host Sponsor is required in this context. 
Why are you asking?

Comment (phrawzty) [inline site/content/events/2016-kiel/_sponsors.txt]: You've got the following (pseudo-code) conditional:

if host_sponsors *is* zero then a_thing
else if host_sponsors is *less than one* then another_thing

The *if zero* should cover the only possible use case, zero being the only non-negative integer that's less than one. :smile: That's why I asked - in order to determine whether the second conditional was set to the wrong integer (i.e. should be less than a larger number), or removed entirely.  The latter case appears to be correct.

Comment (connor21) [inline site/content/events/2016-kiel/_sponsors.txt]: Mmmh, yes. I was a bit lazy, but one Host Sponsor is correct and it's working. If you don't mind I'd leave it as it is.

**Comment by phrawzty** [comment]
Hi, sorry for the delay on this. I decided to try to use Docker to check this PR out locally (it's quite large, so I'd prefer to examine it).  With that yak successfully shaved, I'll go ahead and merge now.

**Comment by connor21** [comment]
Thank you for merging our changes. Have a nice weekend.

**Pull request merged**